web/e2e/page-objects/flows/editFlow.ts

```diff
@@ -43,7 +43,7 @@ export class EditFlow extends AppPage {
   }
 
   get runFlowStatus() {
-    return element(by.css("a.ng-star-inserted"));
+    return element(by.id("latest-job-status"));
   }
```

Comment:
Just the id is matching in 2 places. Locators should always match to exactly 1 we intend to test. Please change this to use by.css(.flow-container #latest-job-status) or by.css(#latest-job-info #latest-job-status) whichever works